packages/block-library/src/gallery/edit.js

```diff
@@ -544,7 +545,7 @@ export default function GalleryEdit( props ) {
 		defaultBlock: DEFAULT_BLOCK,
 		directInsert: true,
 		orientation: 'horizontal',
-		renderAppender: false,
+		renderAppender: InnerBlocks.defaultBlockAppender,
```

Comment:
```suggestion
		renderAppender: InnerBlocks.DefaultBlockAppender,
```

The `DefaultBlockAppender` is a component; currently, it resolves to `undefined`.

Reply:
@Mamaduka Thanks for the correction, also have added before/after screenshots.